src/OrchardCore.Modules/OrchardCore.Twitter/ViewModels/TwitterSettingsViewModel.cs

```diff
@@ -15,5 +15,7 @@ public class TwitterSettingsViewModel
 
         [Required(AllowEmptyStrings = false, ErrorMessage = "Access token secret is required")]
         public string AccessTokenSecret { get; set; }
+
+        public bool CouldNotDecryptSettings { get; set; } = false;
```

Comment:
```suggestion
        public bool CouldNotDecryptSettings { get; set; }
```

And so no need to set false, when that is the default value of a bool